ServerCore/ServerCore.csproj

```diff
@@ -62,6 +62,7 @@
   <ItemGroup>
     <Folder Include="Areas\Identity\Data\" />
     <Folder Include="Content\" />
+    <Folder Include="wwwroot\images\ps24\" />
```

Comment:
Let's remove this - should be unnecessary. Note that other image folders aren't in here.